### ISSUE TITLE
blank cmd window

### ISSUE THREAD

**Issue by ibbizaki123** [Jire/Charlatano#80]
using windows 8.1 when i started cheat first time while i was in-game it downloads many thing and i closed it .. but now i re opened the hack it is showing blank cmd screen .

**Comment by DeathStrax** [comment]
yes, it is supppose to be like that. just wait for the cheat to load in and you are set to cheat in CSGO

**Comment by ibbizaki123** [comment]
i waited 5 hours nothing happens dude :(

**Comment by DeathStrax** [comment]
Hm... for me, when I am in-game, holding a gun, during warm-up, I load in the cheat. Double-click the .bat file. Then I wait for at least 30 secs, then the cheat loads in and I am set to go.

**Comment by ibbizaki123** [comment]
let me try

**Comment by ibbizaki123** [comment]
nothing happened .. same .. and i cant write anything on that cmd window :(

**Comment by FreakyMikey** [comment]
yes, it is supppose to be like that.... like he said before, u have to preload a game what he said being in de warm up and then u can start up the cmd there will be nothing and the cheat will load . All the settings is in the files

**Comment by ibbizaki123** [comment]
may i have some java issue cheat is not working good for me .. i reinstalled java and its sdk again but same :@ idk why it is happening with my pc .. may be jire can help me

**Comment by FreakyMikey** [comment]
Whats the problem ? Isnt it working if u have a game loaded and then u must start de bat file and then they will be activated ? Maybe go to the tutorial??

**Comment by ibbizaki123** [comment]
actually from the very first time when i was updaing the cheat .. it was downloading dependecies and i crossed it because it was taking so much time to downloading them .. and after that i follow the tutorial and i stuck to blank window ..

**Comment by ChristiaanKOch** [comment]
@FreakyMikey 
I have windows 7 and if I trun my cheat on and wanna run it then I get black screen is it no working on the new csgo update can someone please help me

**Comment by ibbizaki123** [comment]
@ChristiaanKOch  @FreakyMikey  add me on discord will provide u my private cheat
ibbi#8002

**Comment by HRVAT007** [comment]
@ibbizaki123 can you provide it to me too?

**Comment by ibbizaki123** [comment]
sure

**Comment by Primeknight19** [comment]
Can you provide me to, just want to see how is it compared to charlatano
